src/node_worker.h

```diff
@@ -34,7 +53,6 @@ class Worker : public AsyncWrap {
     tracker->TrackFieldWithSize(
         "isolate_data", sizeof(IsolateData), "IsolateData");
     tracker->TrackFieldWithSize("env", sizeof(Environment), "Environment");
-    tracker->TrackField("thread_exit_async", *thread_exit_async_);
```

Comment:
You could still track the memory of the new fields (and/or make `LoopStopper` inherit from `MemoryRetainer`)?

Reply:
done, LoopStopper now inherits from MemoryRetainer, and tracks the `async_` field.